docs/_lab/07-writing-plans/index.md

```diff
@@ -49,15 +76,15 @@ Plan completed successfully with no result
 
 **Note:**
 
-* `nodes` is passed as an argument like any other, rather than a flag. This makes plans flexible when it comes to taking lists of different types of nodes or generating the list of nodes in code within the plan.
+* `nodes` is passed as an argument like any other, rather than a flag. This makes plans flexible when it comes to taking lists of different types of nodes or generating the list of nodes in code within the plan. You can still pass the names of groups in the inventory file to this parameter.
```

Comment:
```suggestion
* `nodes` is passed as an parameter like any other, rather than a flag. This makes plans flexible when it comes to taking lists of different types of nodes. You can still pass the names of groups in the inventory file to this parameter.
```